src/Middleware/HeaderPropagation/src/HeaderPropagationState.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.HeaderPropagation
+{
+    public class HeaderPropagationState
+    {
+        private static AsyncLocal<Dictionary<string, StringValues>> _headers { get; } = new AsyncLocal<Dictionary<string, StringValues>>();
```

Comment:
There's no real need for this to be a property.

Reply:
true, corrected! thanks!